libs/emblem/src/components/asset-emblem.test.tsx

```diff
@@ -36,4 +38,20 @@ describe('EmblemByAsset', () => {
       'https://icon.vega.xyz/vega/vega-mainnet-0011/asset/123/logo.svg'
     );
   });
+
+  it('renders a source chain icon by default', () => {
+    const props = {
+      asset: '123',
+    };
+
+    const { getByAltText } = render(<EmblemByAsset {...props} />);
+
+    const chainImage = getByAltText('Chain logo');
+
+    expect(chainImage).toBeInTheDocument();
```

Comment:
This assertion is slightly redundant only because getByAltText will fail if its not in the document

Reply:
Thanks for catching that. I missed this comment, and as we're aiming for a release today I'm going to merge and then fix this issue.